raspberry/bzzz/sensors/gnss.py

```diff
@@ -0,0 +1,186 @@
+import serial
+import numpy as np
+from threading import Thread, Lock
+import time
+import datetime
+from .data_logger import DataLogger
+from .filters import MedianFilter
+
+def deg_min_sec_to_decimal(degrees, minutes, direction):
+    """
+    Convert degrees-minutes-seconds from GNGLL to decimal
+    :param degrees: degrees
+    :param minutes: minutes
+    :param direction: direction (can be one of 'N', 'W', 'E', 'S')
+    :return: decimal representation of GPS coordinates
+    """
+    decimal_degrees = degrees + minutes / 60
+    if direction in ['S', 'W']:  # need to be denoted with a negative value
+        decimal_degrees *= -1
+    return decimal_degrees
+
+class Gnss:
+    """
+    GNSS Module
+
+    This class is used to interface the anemometer
+    """
+    def __init__(self, 
+                 serial_path="/dev/tty.usbmodem14201", 
+                 baud=57600, 
```

Comment:
I think this is correct but could be wrong

I know we have to set the UART 1 to 57600 for RTCM so thought it would be best to keep it the same.

What's everyone else's thoughts?

Reply:
if the rate of correction is 5600 from the base station telemetry I don't think there's any need to go faster

Reply:
The question is what is the baud rate of the serial connection

Reply:
We can configure it with u-centre

Reply:
@alphaville are we happy with this

Reply:
Yep

Reply:
sounds good